io.catenax.week_based_material_demand/1.0.0/WeekBasedMaterialDemand.ttl

```diff
@@ -0,0 +1,252 @@
+    bamm:characteristic :BPNL;
+    bamm:exampleValue "BPNL6666666666YY".
+:unitOfMeasure a bamm:Property;
+    bamm:preferredName "Unit of Measure"@en;
+    bamm:description "Unit of measurement for demand quantities."@en;
+    bamm:characteristic :UnitOfMeasure;
+    bamm:exampleValue "KGM".
+:materialNumberCustomer a bamm:Property;
+    bamm:preferredName "Customer Material Number"@en;
+    bamm:description "Material identifier as assigned by customer. This material number identifies the material (as planned) in customer's database. Must be unique for each Material Demand in the customer-supplier relationship."@en;
+    bamm:characteristic :MaterialNumber;
+    bamm:exampleValue "MNR-7307-AU340474.002".
+:materialNumberSupplier a bamm:Property;
+    bamm:preferredName "Supplier Material Number"@en;
+    bamm:description "Material identifier as assigned by supplier. This material number identifies the material (as planned) in supplier's database."@en;
+    bamm:characteristic :MaterialNumber;
+    bamm:exampleValue "MNR-8101-ID146955.001".
+:materialDescriptionCustomer a bamm:Property;
+    bamm:preferredName "Customer Material Description"@en;
+    bamm:description "Description of the material."@en;
+    bamm:characteristic :MaterialDescription;
+    bamm:exampleValue "Spark Plug".
+:changedAt a bamm:Property;
+    bamm:preferredName "Changed At"@en;
+    bamm:description "Point in time when the material demand was last updated at the customer, either by a human user or an automated process."@en;
+    bamm:characteristic :Timestamp;
+    bamm:exampleValue "2023-03-10T12:27:11.320Z"^^xsd:dateTimeStamp.
+:UUIDv4 a bamm:Characteristic;
+    bamm:preferredName "UUIDv4"@en;
+    bamm:description "A version 4 UUID is a universally unique identifier that is generated using random 32 hexadecimal characters."@en;
+    bamm:dataType xsd:string;
+    bamm:see <http://handle.itu.int/11.1002/1000/11746>.
+:DemandSeriesSet a bamm-c:Set;
+    bamm:preferredName "Demand Series Set"@en;
+    bamm:description "A set of demand series belonging to a certain material demand. Each demand series in the set must have a unique combination of customer location and demand category."@en;
+    bamm:dataType :DemandSeries.
+:BPNL a bamm:Characteristic;
+    bamm:preferredName "Business Partner Number Legal Entity"@en;
+    bamm:description "BPNL represents the legal entity of an organization participating in Catena-X."@en;
+    bamm:dataType xsd:string.
+:expectedSupplierLocation a bamm:Property;
+    bamm:preferredName "Expected Supplier Location"@en;
+    bamm:description "The BPNS number of the site from where the customer expects the supplier to fulfill the demands of the demand series. The BPNS number of the site from where the customer expects the supplier to fulfill the demands of the demand series. The value is used for informational purposes only and is therefore not binding for the supplier."@en;
```

Comment:
@LukasHeimann the description of the `expectedSupplierLocation` property is redundant. Can you fix that as well before the model is submitted for final approval? Thx!

Reply:
Good catch, was a mistake during copy-paste from the email

Reply:
Always happy to review every single bit and byte 😇